.gitpod.yml

```diff
@@ -34,6 +34,7 @@ tasks:
   - name: Go
     init: leeway exec --filter-type go -v -- go get -v ./...
     openMode: split-right
+privileged: true
```

Comment:
We no longer support this in workspace config.
If you want to use this feature flag, please set it on your user.